hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java

```diff
@@ -270,8 +271,9 @@ final HapiSpec payerRecordCreationSanityChecks() {
                     final var payerId = spec.registry().getAccountID(PAYER);
                     final var subOp = getAccountRecords(PAYER).logged();
                     allRunFor(spec, subOp);
-                    final var records =
-                            subOp.getResponse().getCryptoGetAccountRecords().getRecordsList();
+                    final var records = subOp.getResponse().getCryptoGetAccountRecords().getRecordsList().stream()
```

Comment:
This change seems unrelated.  Was this supposed to be part of a separate PR?

Reply:
It can be unrelated but we fix the flaky test here as well. I can change the description